content/developers/api-reference/events-api/index.md

```diff
@@ -86,6 +86,174 @@ The response is:
   "transaction_id": "0x07569"
 }
 ```
+
+### Document Profile Event Creation
+There are two [Document Profile Events](/developers/developer-patterns/document-profile/) that are available as part of the document lifecycle. These are to `publish` a new version and to `withdraw` the document from use.
+
+#### Publish
+Define the Event parameters and store in `/path/to/jsonfile`:
+
+```json
+{
+    "behaviour": "RecordEvidence",
+    "operation": "Record",
+    "asset_attributes": {
+        "document_hash_value":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+        "document_hash_alg":"sha256",
+        "document_status": "Published",
+        "document_version":"2"
+    },
+    "event_attributes": {
+        "arc_description":"Publish version 2 of Test Document",
+        "arc_display_type":"Publish",
+        "document_version_authors": [
+                        {
+                "display_name": "George",
+                "email": "george@rainbow.tv"
```

Comment:
I'm not familiar with the rainbow.tv domain. Is that one we consistently use? Is it a reserved fake one, similar to example.com?

Reply:
It's taken from Jon's Postman collection example and is entirely fictional.

George, Zippy and Bungle were characters from a kid's TV program called Rainbow from the 70s and 80s.

Reply:
Cool: https://en.wikipedia.org/wiki/Rainbow_(TV_series)